tasks/main.yml

```diff
@@ -0,0 +1,64 @@
+---
+
+###
+### Main entrypoint
+###
+
+- name: Check if Helm binary exists
+  shell: command -v helm >/dev/null 2>&1
+  register: helm_exist
+  ignore_errors: yes
+
+- name: Fail if Helm command does NOT exist
+  fail:
+    msg: "Helm command doesn NOT exist. Please install Helm v{{ helm_version }}"
+  when: helm_exist.rc != 0
+
+- name: Get Helm verson
+  command: "helm version --client --template '{{ '{%' }} raw %}{{ '{{' }} .Client.SemVer }}{{ '{%' }} endraw %}'"
+  register: helm_local_version
+  ignore_errors: yes
+
+- name: Fail if Helm version does NOT match
+  fail:
+    msg: "Local Helm version does NOT match the requirements. Please install Helm v{{ helm_version }}"
+  when: "'v{{ helm_version }}' != '{{ helm_local_version.stdout }}'"
```

Comment:
I suggest using something a bit more robust like the [version_compare](https://docs.ansible.com/ansible/2.4/playbooks_tests.html#version-comparison) filter which is built precisely for this and supports several options that might come in hand here as well.

Reply:
If we wish to use `> 1.1` then it would make sense. If we need this, I suggest we attack it on a separate PR to keep things simple.